drivers/gles2/rasterizer_scene_gles2.cpp

```diff
@@ -2310,6 +2310,11 @@ void RasterizerSceneGLES2::_render_render_list(RenderList::Element **p_elements,
 
 			if (accum_pass) { //accum pass force pass
 				blend_mode = RasterizerStorageGLES2::Shader::Spatial::BLEND_MODE_ADD;
+				if (rebind_light && light && light->light_ptr->negative) {
```

Comment:
This looks strange, where does it get reset?

Reply:
In the line above. 

``blend_mode`` is redefined for every object. If it happens to be an ``accum_pass`` the default blend mode is overridden to ``BLEND MODE ADD`` in 2311. All I did was check to see if its a negative light and change that override to be ``BLEND_MODE_SUB``

Reply:
oh i see, makes sense, seems fine then